### PR TITLE
Set up structure

### PR DESCRIPTION
**Pull request by RamazArtmeladze**

### CODE REVIEW

**Review by s-stepanov — COMMENTED** [review]
Looks good overall

Let's modify Readme file a little bit. Now it is to general and it is not fully clear how should someone set up the database (which database? what version?). Should the migrations be run, if yes, then how? Imagine someone who sees the project at a first time - they should be able to set it up step by step

File: src/main/java/com/app/eventOrganizer/controller/TestController.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+@RestController
+@RequiredArgsConstructor
+public class TestController {
+
+    private final TestService testService;
+
+    @PostMapping("/test")
+    public TestModel createTestModel ( @RequestParam Long ID, @RequestParam String name) {
+        return testService.saveToDB(ID, name);
+    }
```
Comment (s-stepanov) [inline src/main/java/com/app/eventOrganizer/controller/TestController.java:18]: Let's change it to /health endpoint. Also creating a record in the database just to check the connection might be too much. Usually `/health` endpoints are used by tools that trigger the endpoint automatically. So, with your implementation, we might come up with lots of data generated in the database. Doing a simple select operation is better for this. Also you can look into something like: https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html for /health endpoint

Comment (RamazArtmeladze) [reply src/main/java/com/app/eventOrganizer/controller/TestController.java]: done

File: src/main/java/com/app/eventOrganizer/controller/TestController.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+@RestController
+@RequiredArgsConstructor
+public class TestController {
+
+    private final TestService testService;
+
+//    @PostMapping("/test")
+//    public TestModel createTestModel ( @RequestParam Long ID, @RequestParam String name) {
+//        return testService.saveToDB(ID, name);
+//    }
```
Comment (s-stepanov) [inline src/main/java/com/app/eventOrganizer/controller/TestController.java:20]: Please remove commented out code

**Review by s-stepanov — APPROVED** [review]

**Pull request merged**